pkgs/tools/graphics/wdisplays/default.nix

```
@@ -1,36 +1,25 @@
-{ lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, gtk3, epoxy, wayland, wrapGAppsHook
-, fetchpatch
-}:
+{ lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, gtk3, epoxy, wayland, wrapGAppsHook }:
 
 stdenv.mkDerivation rec {
-  pname = "wdisplays";
-  version = "1.0";
+  pname = "wdisplays-unstable";
+  version = "2021-04-03";
```

```suggestion
  pname = "wdisplays";
  version = "unstable-2021-04-03";
```
there is only one wdisplays in nixpkgs.